packages/slate-react/src/components/content.js

```diff
@@ -237,7 +259,12 @@ class Content extends React.Component {
 
     // If the Slate selection is unset, but the DOM selection has a range
     // selected in the editor, we need to remove the range.
-    if (selection.isUnset && rangeCount && this.isInEditor(anchorNode)) {
+    if (
+      selection.isUnset &&
+      rangeCount &&
+      this.isInEditor(anchorNode) &&
+      selectionsEqual(this.nativeSelection, native)
```

Comment:
Since this is only used once, can you inline this logic?

Reply:
Also, can you add a comment to explain what is going on there by comparing the current native selection to a cached value?